gantt-chart/.cypress/integration/ui.spec.js

```diff
@@ -5,35 +5,31 @@
 
 /// <reference types="cypress" />
 
-import {
-  testDataSet,
-  delay,
-  GANTT_VIS_NAME,
-  Y_LABEL,
-  X_LABEL,
-  DEFAULT_SIZE,
-} from '../utils/constants';
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+import dayjs from 'dayjs';
```

Comment:
are we moving to dayjs or just for testing?

Reply:
This is just for testing time formats. We had issues earlier when checking absolute time, hence moved to testing time format. FTR has this issue already fixed in.